Bundle/I18nBundle/Resolver/LocaleResolver.php

```diff
@@ -54,15 +54,27 @@ public function resolve(Request $request)
     /**
      * @param Request $request
      *
+     * @throws \Exception
      * @return string
      *
-     * resolves the locale from host
+     * resolves the locale from httpHost or host
      */
     public function resolveFromDomain(Request $request)
     {
-        $host = $request->getHttpHost();
+        $host = $request->getHost();
+        $httpHost = $request->getHttpHost();
+
+        if (array_key_exists($host, $this->localeDomainConfig)) {
+            return $this->localeDomainConfig[$host];
+        } else if (array_key_exists($httpHost, $this->localeDomainConfig)) {
+            return $this->localeDomainConfig[$httpHost];
+        }
+
+        throw new \Exception(sprintf(
+            'Host "%s" is not defined in your locale_pattern_table in app/config/victoire_core.yml',
```

Comment:
You assume the config file is located in app/config/victoire_core.yml but it will be in config.yml most of the time. I'd rather such a message :  

- Host %s is not defined in your victoire_core.locale_pattern_table config. Please check https://github.com/Victoire/victoire/tree/master/Bundle/I18nBundle#configuration-type for more information.